anna/my-app/src/pages/movieDesc/HomePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Layout from '../../components/layout';
+import './homePage.css';
+import MovieList from '../../components/movieList';
+import IMovie from '../../components/movieList/movie-card/IMovie';
+import SearchForm from '../../components/searchForm';
+import { ISearchFormState } from '../../components/searchForm/SearchForm';
+import SortBox from '../../components/sortBox';
+import Container from '../../components/container';
+
+interface IHomeState {
+  movies: Array<IMovie>
```

Comment:
**optional comment**
more modern way to write IMovie[];

Reply:
done